test-data/unit/check-modules.test

```diff
@@ -1312,7 +1312,7 @@ class A:
     def f(self) -> str: return 'foo'
 class B(A):
     def f(self) -> str: return self.x
-    def initialize(self): self.x = 'bar'
+    def initialize(self) -> None: self.x = 'bar'
```

Comment:
Not having a return type may have well been intentional. I'd prefer either keeping the original test case or adding another variant with a return type annotation.